src/segment_owd_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestOWDStringPlayingSong(t *testing.T) {
+	expected := "滛 (16.7°)"
+	d := &owm{
+		temperature: 16.7,
+		weather:     "滛",
+	}
+
+	assert.Equal(t, expected, d.string())
+}
+func bootstrapOWDTest(json string, err error) *owm {
```

Comment:
We moved to another style of tests to facilitate adding use cases. See [here](https://github.com/JanDeDobbeleer/oh-my-posh/blob/572149b176b864775edae0e531ed38153a12d66a/src/segment_battery_test.go#L17) for example.

Reply:
Updated in c547971